### PR TITLE
HCAP-879 | Confirm success spacing adjustment

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-879

image
image
image

### CODE REVIEW

File: client/src/pages/private/ParticipantEOI.js [hunk 1]
```diff
@@ -267,55 +267,50 @@ export default () => {
...
-                isDisabled={!enableEdit}
-                hideSummary={true}
-                showIdentityQuestions
-                enableFields={['phoneNumber', 'postalCode']}
-                onSubmit={onFormSubmit}
-                editMode={true}
-                isSubmitted
-              />
-            </Grid>
+          <Grid item xs={12} sm={11} md={10} lg={8} xl={6}>
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantEOI.js:270]: I think, there will be possible conflict with PR-#577

**Review by fwpushan — COMMENTED** [review]
Technically all good please check my comment

File: client/src/pages/private/ParticipantEOI.js (hunk 1 above)
Comment (arranfw) [reply client/src/pages/private/ParticipantEOI.js:270]: This PR is branched off that one so it should work out

Comment (arranfw) [reply client/src/pages/private/ParticipantEOI.js]: Ah I had the wrong base, my bad

**Review by fwpushan — APPROVED** [review]

**Comment by arranfw** [comment]
Test fixed

**Review by fwpushan — APPROVED** [review]

**Pull request merged**